### ISSUE TITLE
Fix Fastlane for iOS

### ISSUE THREAD

**Issue by bartonhammond** [bartonhammond/snowflake#26]

**Comment by bartonhammond** [comment]
Must be careful about deploying to iTunes/AppStore.

**Comment by bartonhammond** [comment]
It's probably because at one time I was submitting the app to the App Store and when that functionality was removed, the config files were missed.  

What's missing/wrong w/ the Fastlane for iOS is the whole submission process - all that is being done now is testing and then building the app.  

At the time I wrote this, and still true, I didn't have a lot of understanding of the Apple App Store submission process, the requirments, bundling, certificates, etc. I initially was going to submit the app but couldn't figure out what all I needed to do so I punted and put together a starting point from which others could build off of. 

Thanks for looking at Snowflake.
